### PR TITLE
v0.8.0 Preflight 0.1

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Pullflow is very cool @henry-richard 
Screen Shot 2022-01-06 at 2 57 26 PM

**Comment by wanda-carlson** [comment]
You should try it **Henry Richard**

**Comment by henry-richard** [comment]
Oh, it posted from me! @wanda-carlson

File: src/wanda.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 print("this is wanda")
 print('this is wanda too')
 print("Wanda testing v0.8.0")
```
Comment (henry-richard) [inline src/wanda.py:3]: was this for local?

File: src/wanda.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+print("Wanda testing v0.8.0 Preflight 0.1")
```
Comment (henry-richard) [inline src/wanda.py:4]: why the 0.1?

**Review by henry-richard — CHANGES_REQUESTED** [review]
Can we get explanatory comments?

**Comment by wanda-carlson** [comment]
0.1 is because I dont know how many i will need inorder to test all the cases.

File: src/wanda.py (hunk 1 above)
Comment (wanda-carlson) [reply src/wanda.py:3]: Yes

**Review by henry-richard — APPROVED** [review]

**Pull request merged**